resources/assets/less/bem/profile-links.less

```diff
@@ -36,4 +42,27 @@
     font-weight: 600;
     word-wrap: break-word;
   }
+
+  .edit-profile-links {
```

Comment:
this should conform with [bem](https://css-tricks.com/bem-101/). Either as a new block (in its own file) or as element of `profile-links` (`profile-links__actions`)